ui/components/app/modals/deposit-ether-modal/index.scss

```diff
@@ -109,7 +109,7 @@
       padding-bottom: 20px;
       align-self: flex-start;
 
-      @media screen and (min-width: 575px) {
+      @media screen and (min-width: $break-small) {
```

Comment:
Is this one and the one right below it correct? Shouldn't it be `min-width: $break-large` or `max-width: $break-small`?

Reply:
Fixed